docker-compose.yml

```diff
@@ -8,16 +13,69 @@ services:
       - ./data/db:/data
       - ./config/virtuoso/virtuoso.ini:/data/virtuoso.ini
       - ./config/virtuoso/toLoad:/data/toLoad
+    labels:
+      - "logging=true"
+    restart: always
+  database:
+    image: semtech/sparql-parser:0.0.13
+    volumes:
+      - ./config/authorization:/config
+      - ./data/authorization:/data
+    labels:
+      - "logging=true"
+    restart: always
   migrations:
     # Publiq data loaded through Virtuoso toLoad because of its size
     image: semtech/mu-migrations-service:0.9.0
     environment:
       MU_SPARQL_ENDPOINT: "http://virtuoso:8890/sparql"
     volumes:
       - ./config/migrations:/data/migrations
+    labels:
+      - "logging=true"
+    restart: always
   entity-mapping:
     image: redpencil/publiq-entity-mapping-service:latest
     environment:
       WORKERS: "1"
       MU_SPARQL_ENDPOINT: "http://virtuoso:8890/sparql"
       MU_SPARQL_UPDATEPOINT: "http://virtuoso:8890/sparql"
+    labels:
+      - "logging=true"
+    restart: always
+  identifier:
+    image: semtech/mu-identifier:1.10.3
+    environment:
+      SESSION_COOKIE_SECURE: "on"
+    links:
+      - dispatcher:dispatcher
+    labels:
+      - "logging=true"
+    restart: always
+  dispatcher:
+    image: semtech/mu-dispatcher:2.1.0-beta.2
+    links:
+      - resource:resource
+    volumes:
+      - ./config/dispatcher:/config
+    labels:
+      - "logging=true"
+    restart: always
```

Comment:
Put these services on top of the file to "mimic" the flow of requests  identifier --> dispatcher --> frontend / other service. Same in `docker-compose.dev.yml`.